europy/lifecycle/modeldetails.py

```diff
@@ -0,0 +1,48 @@
+from datetime import datetime
+from typing import List, Optional
+
+class ModelDetails:
```

Comment:
ditto with the file name, should it be `model_details`

Reply:
👍